app/routes/circulars/circulars.server.ts

```diff
@@ -172,3 +173,18 @@ export async function put(subject: string, body: string, request: Request) {
     submitter: formatAuthor(user),
   })
 }
+
+export async function circularRedirect(query: string) {
+  const validCircularSearchStyles =
+    /^\s*(?:GCN)?\s*(?:CIRCULAR)?\s*(-?\d+(?:\.\d)?)\s*$/i
+  const circularId = parseFloat(
+    validCircularSearchStyles.exec(query)?.[1] || ''
+  )
+  if (!isNaN(circularId)) {
+    const db = await tables()
+    const result = await db.circulars.get({ circularId })
+    if (!result) return
+    const circularURL = `/circulars/${circularId}`
+    throw redirect(circularURL, { status: 302 })
```

Comment:
According to https://remix.run/docs/en/main/utils/redirect, 302 is the default redirect status code, so we can omit it.

```suggestion
    throw redirect(circularURL)
```